### DISCUSSION TITLE
PID control for Coffed SR15

### DISCUSSION THREAD

**Discussion by swroasting** [artisan-roaster-scope/artisan#1431] [Q&A]
I have my Coffed SR15 connected to Artisan (2.10.0 c44a180) and I can control the roaster manually, but I cannot figure out how to get the PID tuned.  It seems to not change the control value until after it crosses the set value, and it makes very large changes and holds them until it crosses the set value again.  I have tried all kinds of numbers and this is the best I can get.  Any ideas what I'm doing wrong?
![sr15pid](https://github.com/artisan-roaster-scope/artisan/assets/158624484/90fea5eb-dc0c-4a14-8937-cee2c8dc72df)
Also, how do I attach my aset file, github says its not a supported file type?

**Comment by poundy** [comment]
you have to rename the file as a .txt file and then upload it

**Comment by swroasting** [comment]
Aset attached here
[SR15 Aset Txt.txt](https://github.com/artisan-roaster-scope/artisan/files/14147617/SR15.Aset.Txt.txt)
